src/constants/ow1_constants.opy

```diff
@@ -164,6 +165,7 @@
 #!define OW1_ORISA_SUPERCHARGER_HEIGHT 1
 #!define OW1_ORISA_SUPERCHARGER_LINGER_TIME 1
 #!define OW1_ORISA_SUPERCHARGER_MAX_DURATION 15
+#!define OW1_ORISA_SHIELDS 0
```

Comment:
Can you reorder the constants so they're alphabetical for each hero?

Reply:
I can also reorder it myself if you want to focus on something else.

Reply:
Done